src/Compilers/CSharp/Portable/Syntax/SyntaxFactory.cs

```diff
@@ -1366,17 +1366,8 @@ public static SyntaxTriviaList TriviaList(params SyntaxTrivia[] trivias)
         /// <param name="trivias">A sequence of trivia.</param>
         public static SyntaxTriviaList TriviaList(IEnumerable<SyntaxTrivia> trivias)
         {
-            if (trivias == null)
-            {
-                return default(SyntaxTriviaList);
-            }
```

Comment:
Aren't we allocating a `SyntaxTriviaListBuilder` in the `trivias == null` case now?